js/utils/taskCountHandler.js

```
@@ -0,0 +1,43 @@
+    case "delete": {
+      allTasksCountElement.textContent =
+        Number(allTasksCountElement.textContent) - 1;
+
+      if (task.completed) {
+        completedTasksCountElement.textContent =
+          Number(completedTasksCountElement.textContent) - 1;
+      } else {
+        pendingTasksCountElement.textContent =
+          Number(pendingTasksCountElement.textContent) - 1;
+      }
+    }
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)